src/sentry/options/defaults.py

```diff
@@ -19,7 +19,21 @@
 register('system.debug', default=False, flags=FLAG_NOSTORE)
 register('system.rate-limit', default=0, flags=FLAG_PRIORITIZE_DISK)
 register('system.secret-key', flags=FLAG_NOSTORE)
-register('redis.options', type=Dict, flags=FLAG_NOSTORE)
+register(
+    'redis.clusters',
+    type=Dict,
+    default={
+        'default': {
+            'hosts': {
+                0: {
+                    'host': '127.0.0.1',
+                    'port': 6379,
```

Comment:
These defaults are unnecessary, but I just copied over the value of `SENTRY_REDIS_OPTIONS`.